src/Executables/Atlas/3. Configuration/2. Advanced Configuration/Utilities/Process Explorer/Uninstall Process Explorer (default).cmd

```diff
@@ -6,6 +6,8 @@ whoami /user | find /i "S-1-5-18" > nul 2>&1 || (
 	exit /b
 )
 
+reg delete "HKCU\SOFTWARE\Sysinternals\Process Explorer" /f
```

Comment:
Maybe ask if they want to delete Process Explorer settings?

Reply:
Re solved in https://github.com/Atlas-OS/Atlas/pull/797/commits/a403bec5d0aa187fa05e3edfd5c6f7ad92b5d2bb.